waku/v2/protocol/waku_rln_relay/waku_rln_relay_metrics.nim

```diff
@@ -31,3 +42,48 @@ declarePublicGauge(waku_rln_registration_duration_seconds, "time taken to regist
 declarePublicGauge(waku_rln_instance_creation_duration_seconds, "time taken to create an rln instance")
 declarePublicGauge(waku_rln_membership_insertion_duration_seconds, "time taken to insert a new member into the local merkle tree")
 declarePublicGauge(waku_rln_membership_credentials_import_duration_seconds, "time taken to import membership credentials")
+
+template parseAndAccumulate(collector: Collector, cumulativeValue: float64): float64 =
+  let total = parseCollectorIntoF64(collector)
+  let freshCount = max(total - cumulativeValue, 0)
+  cumulativeValue = total
+  freshCount
+
+proc logRlnMetrics*() =
+  var logMetrics: proc(udata: pointer) {.gcsafe, raises: [Defect].}
+
+  var cumulativeErrors = 0.float64
+  var cumulativeMessages = 0.float64
+  var cumulativeSpamMessages = 0.float64
+  var cumulativeInvalidMessages = 0.float64
+  var cumulativeValidMessages = 0.float64
+  var cumulativeProofs = 0.float64
+
+  logMetrics = proc(udata: pointer) =
+    {.gcsafe.}:
+
+      let freshErrorCount = parseAndAccumulate(waku_rln_errors_total,
+                                               cumulativeErrors)
+      let freshMsgCount = parseAndAccumulate(waku_rln_messages_total,
+                                             cumulativeMessages)
+      let freshSpamCount = parseAndAccumulate(waku_rln_spam_messages_total,
+                                              cumulativeSpamMessages)
+      let freshInvalidMsgCount = parseAndAccumulate(waku_rln_invalid_messages_total,
+                                                    cumulativeInvalidMessages)
+      let freshValidMsgCount = parseAndAccumulate(waku_rln_valid_messages_total,
+                                                  cumulativeValidMessages)
+      let freshProofCount = parseAndAccumulate(waku_rln_proof_verification_total,
+                                               cumulativeProofs)
+
+      
+      info "Total messages", count = freshMsgCount
+      info "Total spam messages", count = freshSpamCount
+      info "Total invalid messages", count = freshInvalidMsgCount
+      info "Total valid messages", count = freshValidMsgCount
+      info "Total errors", count = freshErrorCount
+      info "Total proofs verified", count = freshProofCount
+
+    discard setTimer(Moment.fromNow(LogPeriod.seconds), logMetrics)
+  
+  logRlnMetrics()
```

Comment:
Shouldn't it be `logMetrics()`?

Reply:
Addressed in [eb9101f](https://github.com/status-im/nwaku/pull/1262/commits/eb9101ff1b19478b500921cbb93433e2f140071c)